python/kfserving/kfserving/kfserver.py

```diff
@@ -126,8 +136,52 @@ def get(self, name: str):
 
 
 class ListHandler(tornado.web.RequestHandler):
-    def initialize(self, models: Dict[str, KFModel]):
+    def initialize(self, models: KFModelRepository):
         self.models = models  # pylint:disable=attribute-defined-outside-init
 
     def get(self):
-        self.write(json.dumps(list(self.models.values())))
+        self.write(json.dumps([ob.name for ob in self.models.get_models()]))
+
+
+class LoadHandler(tornado.web.RequestHandler):
+    def initialize(self, models: KFModelRepository):  # pylint:disable=attribute-defined-outside-init
+        self.models = models
+
+    async def post(self, name: str):
+        try:
+            loop = asyncio.get_running_loop()
+            ready = loop.run_until_complete(self.models.load(name))
```

Comment:
I thought we wanted to make this async and use await here.

Reply:
I prefer to get the future till it is finished to ensure model is loaded successfully

Reply:
@wengyao04 effectively await does that and here probably that's also how await is implemented under the hood, alibi uses that mainly because it is calling a async predict in the sync explainer handler.